util/mock/store.go

```diff
@@ -80,3 +80,8 @@ func (*Store) GetMinSafeTS(_ string) uint64 {
 func (*Store) GetLockWaits() ([]*deadlockpb.WaitForEntry, error) {
 	return nil, nil
 }
+
+// GetCodec implements kv.Storage interface.
+func (*Store) GetCodec() tikv.Codec {
```

Comment:
Where is this used?

Reply:
It's implements for Storage interface.
If I don't implements this function, compile process will meet error:`*mock.Store does not implement "github.com/pingcap/tidb/kv".Storage (missing GetCodec method)`

Reply:
https://github.com/pingcap/tidb/pull/40269/files#diff-96b53c3899fc6766a7004ec8f5ca22f665022b700f47400fbdbd2d100a1b9443R655

That's because you add it here.
If it's not actually used, please avoid the useless method.

Reply:
It's used in https://github.com/pingcap/tidb/blob/6848370005a94300400cf04ef725bf40cf2dc80d/domain/domain.go#L971